scrapinghub/client/exceptions.py

```diff
@@ -81,6 +81,11 @@ def wrapped(*args, **kwargs):
             elif exc._type == APIError.ERR_AUTH_ERROR:
                 raise Unauthorized(http_error=exc)
             raise ScrapinghubAPIError(msg)
+        except ValueError as exc:
+            # exception on attempt to decode wrong json in _decode_response
+            raise ScrapinghubAPIError(
+                "Error decoding server response ({}). Please try again later.".
+                format(str(exc)))
```

Comment:
I don't think it's a good idea to catch all ValueErrors here and convert them to ScrapinghubAPIError. This is specifically a problem of legacy python-scrapinghub, we can fix it there, where it happens - e.g. by checking response code before trying to get json and raising and APIError. I Even if this will change behaviour - not a big deal, this is rather bug than a feature of legacy code. And I think that the new client desires major version bump, so we can allow ourselves to have some small incompatibilities in v2.0.0